concent_api/core/tasks.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from celery import shared_task
+
+from conductor.models import VerificationRequest
+from verifier.tasks import blender_verification_order
+
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task
+def upload_finished(subtask_id: str):  # pylint: disable=unused-argument
```

Comment:
@dybi In #413

Reply:
It's #413. We don't handle it yet here.

@rwrzesien Please remove the `pylint: disable`. The warning is here because the code is incomplete and it's right. Let's not suppress it. It'll go away when we have it fully implemented. This branch can't be deployed on its own anyway.

Reply:
:+1: